node/src/chain_spec.rs

```diff
@@ -125,6 +139,100 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
+                .to_vec(),
+            changes_trie_config: Default::default(),
+        },
+        pallet_aura: AuraConfig {
+            authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
+        },
+        pallet_grandpa: GrandpaConfig {
+            authorities: initial_authorities
+                .iter()
+                .map(|x| (x.1.clone(), 1))
+                .collect(),
+        },
+        pallet_balances: vanilla_runtime::BalancesConfig {
+            balances: endowed_accounts
+                .iter()
+                .cloned()
+                .map(|k| (k, 1 << 60))
+                .collect(),
+        },
+        pallet_sudo: vanilla_runtime::SudoConfig { key: root_key },
+        orml_tokens: vanilla_runtime::TokensConfig {
+            endowed_accounts: endowed_accounts
+                .iter()
+                .flat_map(|x| {
+                    vec![
+                        (x.clone(), CurrencyId::DOT, 1_000 * TOKEN_DECIMAL),
+                        (x.clone(), CurrencyId::KSM, 1_000 * TOKEN_DECIMAL),
+                        (x.clone(), CurrencyId::BTC, 1_000 * TOKEN_DECIMAL),
+                        (x.clone(), CurrencyId::USDC, 1_000 * TOKEN_DECIMAL),
+                        (x.clone(), CurrencyId::xDOT, 1_000 * TOKEN_DECIMAL),
+                    ]
+                })
+                .collect(),
+        },
+        pallet_loans: vanilla_runtime::LoansConfig {
+            currencies: vec![
+                CurrencyId::DOT,
+                CurrencyId::KSM,
+                CurrencyId::BTC,
+                CurrencyId::USDC,
+                CurrencyId::xDOT,
+            ],
+            // total_supply: 1000 * TOKEN_DECIMAL, // 1000
+            // total_borrows: 600 * TOKEN_DECIMAL, // 600
+            borrow_index: RATE_DECIMAL,               // 1
+            exchange_rate: 2 * RATE_DECIMAL / 100,    // 0.02
+            base_rate: 2 * RATE_DECIMAL / 100,        // 0.02
+            multiplier_per_year: RATE_DECIMAL / 10,   // 0.1
+            jump_muiltiplier: 11 * RATE_DECIMAL / 10, // 1.1
+            kink: 8 * RATE_DECIMAL / 10,              // 0.8
+            collateral_rate: vec![
+                (CurrencyId::DOT, 5 * RATE_DECIMAL / 10),
+                (CurrencyId::KSM, 5 * RATE_DECIMAL / 10),
+                (CurrencyId::BTC, 5 * RATE_DECIMAL / 10),
+                (CurrencyId::USDC, 5 * RATE_DECIMAL / 10),
+                (CurrencyId::xDOT, 5 * RATE_DECIMAL / 10),
+            ],
+            liquidation_incentive: vec![
+                (CurrencyId::DOT, 9 * RATE_DECIMAL / 10),
+                (CurrencyId::KSM, 9 * RATE_DECIMAL / 10),
+                (CurrencyId::BTC, 9 * RATE_DECIMAL / 10),
+                (CurrencyId::USDC, 9 * RATE_DECIMAL / 10),
+                (CurrencyId::xDOT, 9 * RATE_DECIMAL / 10),
+            ],
+            //FIXME :In fact,"liquidation_threshold" should be higher than "collateral_rate",
```

Comment:
Can we have a issue to track it, and add more context in the issue?

Reply:
Maybe issue #40 caused by the low liquidation_threshold.